packages/cli/src/main/kotlin/elide/tool/cli/AbstractSubcommand.kt

```diff
@@ -434,17 +462,16 @@ import org.graalvm.polyglot.Engine as VMEngine
 
   /**
    * Run a [block] of code using a [PolyglotContext] configured by the current [engine]. The context is guaranteed to
-   * be exclusive, and it will not be reused after this operation completes.
+   * be exclusive to this thread, but it may be reused after this operation completes.
    *
    * The first invocation of this method will cause the [engine] to be initialized, triggering the
    * [configureEngine] event.
    */
   protected open fun withContext(block: (PolyglotContext) -> Unit) {
```

Comment:
semantics have changed now since the contexts will be reused within the same thread, though there are no behavior changes expected because of this (there's really no context reuse currently outside of server agent code)